cilium/README.md

```diff
@@ -22,7 +22,7 @@ The Cilium check is included in the [Datadog Agent][3] package, but it requires
    
 Or, separately enable Prometheus metrics in the Kubernetes manifests:
 
-   - In the `cilium-agent` add `--prometheus-serve-addr=:9090` to the `args` section of the Cilium DaemonSet config:
+   - In the `cilium-agent` add `--prometheus-serve-addr=:9090` (for [Cilium >= v1.12][14], use port `9962`) to the `args` section of the Cilium DaemonSet config:
```

Comment:
I would update the command line and the yaml example below to use 9092 and leave a note for users who are using Cilium < v1.12. What do you think?

Reply:
@ollieharDD If you decide to add a note, you can either go with bolded **Note**  or with an alert-warning div

**Note**: For [Cilium >= v1.12][14], use port `9962`.

```
  <div class="alert alert-warning">For <a href="https://docs.cilium.io/en/v1.12/operations/upgrade/#id2">Cilium >= v1.12</a>, use port `9962`.</div>
```

Reply:
Hi @estherk15 and @FlorentClarret 👋 I've gone ahead and updated - let me know what you think. Just one thing. I tried to add the banner between the bullet and the yaml, however in doing that, the yaml format was off. Let me know if this is okay or there needs to be a change !